### DISCUSSION TITLE
Extract graph properties

### DISCUSSION THREAD

**Discussion by makkus** [DHARPA-Project/kiara_plugin.network_analysis#11] [Workflows]
# Description

A very simple, one-step workflow to extract graph properties from network data.

It's questionable whether this workflow needs to be implemented in the first place, the alternative would be include the output that is described here in the network_graph data type metadata.

# Steps

1) Extract properties

Extract generic properties that every graph object has.

- Inputs:
  - graph (type: network_graph)

- Outputs
  - number_of_nodes (type: integer)
  - number_of_edges (type: integer)

**Comment by yaslena** [comment]
I think this issue is now resolved with the implemented "get_property_data" kiara API function (https://dharpa.org/kiara/latest/reference/kiara/api/#kiara.api.Value) which can be called with `kiara data explain -p <alias>` in the CLI. 

It now also provides the respective number of edges for the dfferent graph types:

<img width="699" alt="Screenshot 2023-02-15 at 17 48 01" src="https://user-images.githubusercontent.com/53467834/226915493-aeeecc95-ef18-46d8-9df2-671f5d49cdac.png">
